src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/mha.cpp

```diff
@@ -137,6 +137,32 @@ INSTANTIATE_TEST_SUITE_P(smoke_Snippets_MHAEnforceBF16,
                                             ::testing::Values(CPUTestUtils::cpu_bf16_plugin_config)),
                          MHA::getTestCaseName);
 
+INSTANTIATE_TEST_SUITE_P(smoke_Snippets_MHA_FP16_4D,
+                         MHA,
+                         ::testing::Combine(::testing::ValuesIn(transposedShape_4D(false)),
```

Comment:
Since the PR https://github.com/openvinotoolkit/openvino/pull/27452 has been merged, can we test dynamic input shapes for FP16 case too?

The brief explanation: The PR https://github.com/openvinotoolkit/openvino/pull/27452 added support of Brgemms with `K % VNNI_FACTOR != 0` on AMX platforms. Now we can execute `Brgemm` blocks with any shapes using AMX impl.

Reply:
@a-sidorova , dynamic cases is added. I test on GNR machine that need relax threshold a little bit to pass. I doesn't ignore dynamic shape callback similar as bf16 on release. "IGNORE_CALLBACK" used in test just for functional test.